front/app/src/Views/TheGame/TheGame.tsx

```diff
@@ -1,22 +1,165 @@
-import { useEffect } from "react";
-import MenuBar from "../../Component/MenuBar/MenuBar";
-import { useSavePrevPage } from "../../Router/Hooks/useSavePrevPage";
+/* eslint-disable max-lines-per-function */
+/* eslint-disable max-statements */
+import React, { createContext } from 'react';
+import { useEffect, useRef } from "react";
+import { useAppSelector } from "../../hooks/redux-hooks";
+import UseWindowSize from "../Canvas/UseWindowSize";
 
-const	TheGame = () =>
+import Game from "./objects/Game";
+import Board from "./objects/Board";
+import Ball from "./objects/Ball";
+import Net from "./objects/Net";
+import Player from "./objects/Player";
+import Racket from "./objects/Racket";
+import Position from "./objects/Position";
+import Dimension from "./objects/Dimension";
+import { renderMatches } from 'react-router-dom';
+
+/* docu https://css-tricks.com/using-requestanimationframe-with-react-hooks/*/
+const	GameCanvas = () =>
```

Comment:
We've got a code regression, we lost meta-data from the custom hook useSavePrevPage:
	[...]
	const	savePrevPage = useSavePrevPage();
	[...]
	useEffect(() =>
	{
		savePrevPage("/play-game");
		[...]
	}
	[...]